src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEMethodSymbol.cs

```diff
@@ -1067,12 +1052,12 @@ private MethodKind ComputeMethodKind()
             return MethodKind.Ordinary;
         }
 
-        internal override Microsoft.Cci.CallingConvention CallingConvention
+        internal override Cci.CallingConvention CallingConvention
```

Comment:
This is very common pattern across all Roslyn sources to just use `Cci` qualifier rather than the full `Microsoft.Cci` when we are in a declaration nested somewhere under `Microsoft` namespace.